geometry/TrianglesViewController.swift

```diff
@@ -25,4 +26,10 @@ class TrianglesViewController: UIViewController, TrianglesProtocol {
         super.viewDidLoad()
         (self.view as! TrianglesView).setDataSource(self)
     }
+    
+    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
+        
```

Comment:
Methods with empty body look weird. Please add at least logging with proper explanation of why it's here until code is written.